### PR TITLE
Fixes #18558: Adapt rudder plugins makefile so that we make release based on new versionning scheme

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://issues.rudder.io/issues/18558

### CODE REVIEW

**Review by Fdall — APPROVED** [review]

File: centreon/build.conf [hunk 1]
```diff
@@ -22,7 +22,8 @@ plugin-web-description=<p>Plugin for Centreon integartion with Rudder</p>
 # - x.y(.z): plugin major.minor.micro. Micro should be omitted. When omitted, z is assumed to be 0. 
 # For the build, we split the information between two properties, rudder branch and plugin version, 
 # which must be concaneted with "-" to build the plugin version. 
-plugin-branch=1.5-SNAPSHOT
+plugin-branch=1.5
+plugin-branch-nightly=I.5
```
Comment (Fdall) [inline centreon/build.conf:26]: I -> 1

**Review by Fdall — CHANGES_REQUESTED** [review]

File: centreon/build.conf (hunk 1 above)
Comment (amousset) [reply centreon/build.conf:26]: This, or we decide to switch to roman numerals for nightlies

Comment (amousset) [reply centreon/build.conf:26]: RUDDER VI.II.III

**Comment by VinceMacBuche** [comment]
PR updated with a new commit

**Comment by VinceMacBuche** [comment]
OK, squash merging this PR

**Pull request merged**